targets/zephyr/src/main-zephyr.c

```diff
@@ -135,9 +136,21 @@ static int shell_cmd_handler (int argc, char *argv[])
 
   if (jerry_value_has_error_flag (ret_val))
   {
-    printf ("Failed to run JS\n");
+    /* User-friendly error messages require at least "cp" JerryScript
+       profile. Include a message prefix in case "cp_minimal" profile
+       is used. */
+    printf ("Error executing statement: ");
+    /* Clear error flag, otherwise print call below won't produce any
+       output. */
+    jerry_value_clear_error_flag (&ret_val);
   }
 
+  jerry_value_t ret_val_print = jerry_call_function (print_function,
+    jerry_create_undefined (),
+    &ret_val,
+    1);
```

Comment:
Although a `jerry_release_value` call on an undefined value is a no-op at the moment, this can be changed later. I suggest you to put the value of 'this' into a local variable and call `jerry_release_value` after the function call. The code will be more stable.

```c
jerry_value_t this_val =  jerry_create_undefined ();
jerry_value_t ret_val_print = jerry_call_function (print_function, this, &ret_val, 1);
jerry_release_value (this_val);
```